lms/lmsweb/views.py

```diff
@@ -440,4 +478,4 @@ class AdminModelView(AccessibleByAdminMixin, ModelView):
 )
 
 for m in ALL_MODELS:
-    admin.add_view(AdminModelView(m))
+    admin.add_view(SPECIAL_MAPPING.get(m, AdminModelView)(m))
```

Comment:
What about all the other mappings?

Reply:
I put the default, to override only for specific modules the admin view